packages/graphql/src/schemaTypes/objectTypes/gql-Migration.ts

```diff
@@ -0,0 +1,55 @@
+import { MIGRATION_STEPS } from '@packages/types'
+import { enumType, objectType } from 'nexus'
+
+export const MigrationStepEnum = enumType({
+  name: 'MigrationStepEnum',
+  members: MIGRATION_STEPS,
+})
+
+// TODO: implement these values for migration using the ctx
+export const Migration = objectType({
+  name: 'Migration',
+  description: 'The Wizard is a container for any state associated with initial onboarding to Cypress',
```

Comment:
Should be a description of the Migration objectType

Reply:
```suggestion
  description: 'The Migration contains will contain all data related to the 9.X to 10.0 migration UI',
```